src/super_gradients/training/pipelines/pipelines.py

```
@@ -377,3 +379,64 @@ def _combine_image_prediction_to_video(
+        """Decode the model output
+
+        :param model_output:    Direct output of the model, without any post-processing.
+        :param model_input:     Model input (i.e. images after preprocessing).
+        :return:                Predicted Bboxes.
+        """
+        confidence_predictions, classifier_predictions = torch.max(model_output, 1)
+
+        classifier_predictions = classifier_predictions.detach().cpu().numpy()
+        # todo do we need this?
+        confidence_predictions = confidence_predictions.detach().cpu().numpy()
+
+        predictions = list()
+        for prediction, confidence, image_input in zip(classifier_predictions, confidence_predictions, model_input):
+            predictions.append(ClassificationPrediction(confidence=confidence, labels=prediction, image_shape=image_input.shape))
+        return predictions
+
+    def _instantiate_image_prediction(self, image: np.ndarray, prediction: ClassificationPrediction) -> ImagePrediction:
+        return ImageClassificationPrediction(image=image, prediction=prediction, class_names=self.class_names)
+
+    def _combine_image_prediction_to_images(
+        self, images_predictions: Iterable[ImageClassificationPrediction], n_images: Optional[int] = None
+    ) -> ImagesClassificationPrediction:
+        if n_images is not None and n_images == 1:
+            # Do not show tqdm progress bar if there is only one image
+            images_predictions = [next(iter(images_predictions))]
+        else:
+            images_predictions = [image_predictions for image_predictions in tqdm(images_predictions, total=n_images, desc="Predicting Images")]
+
+        return ImagesClassificationPrediction(_images_prediction_lst=images_predictions)
+
+    def _combine_image_prediction_to_video(
+        self, images_predictions: Iterable[ImageDetectionPrediction], fps: float, n_images: Optional[int] = None
+    ) -> VideoDetectionPrediction:
+        # TODO we dont need this in classification task
```

In CalssifiocationPipeline we dont need to implement this function. what should i code their?

---

Maybe something like `raise NotImplementedError('This feature is not available for Classification task')` ?